{
  "type": "class",
  "name": "MinecraftException",
  "qualified_name": "net.minecraft.world.MinecraftException",
  "package": "net.minecraft.world",
  "modifiers": [],
  "annotations": [],
  "description": "",
  "inheritance": {
    "extends": null,
    "implements": [],
    "inheritance_tree": []
  },
  "nested_classes": [],
  "fields": [],
  "constructors": [
    {
      "name": "MinecraftException",
      "modifiers": [],
      "parameters": [],
      "description": "",
      "deprecated": false
    }
  ],
  "methods": [],
  "enum_constants": [],
  "since": null,
  "deprecated": false,
  "source_file": "Frameworks/1.7.10/net/minecraft/world/MinecraftException.html"
}